File: C Programming/Factorial.c
#include<stdio.h>
#include<conio.h>
int main()
{
   long int i,n,sum = 1;
    
    printf("Enter the Number : ");
    scanf("%d",&n);

    for(i=1;i<=n;i++)
    {
        sum =sum *i;
    }

    printf("The Factorial of %ld is %ld ",n,sum);

    getch();
    return 0;
}

File: Java Programming/Factorial.java
class Factorial 
{
    public static void main(String [] rgs)
    {
        int i,n=5,sum=1;
        System.out.println("The number is "+n);
        for(i=1;i<=n;i++)
        {
            sum*=i;
        }
        System.out.println("The Factorial is "+sum);
    }    
}
